apps/docs/content/features/infrastructure.mdx

```diff
@@ -2,13 +2,13 @@
 title: Project & Services Structure
 description: Explore Project & Services Structure on Zerops.
 ---
-Zerops structure has three levels - _projects_, _services_ and _containers_.
+Zerops structure has three levels - **projects**, **services** and **containers**.
 
 ## Project
 
-Project is the top-level entity, which consists of **services**. TODO max amount of projects
+Project is the top-level entity, which consists of **services**. {/* TODO max amount of projects */}
 
-To function properly, a project needs its **"core"** to be running - which contains a logger service, a statistics service, HTTP routing with SSL certificates, IP routing with built-in firewall etc. It is created when the first user service is deployed and is charged a fixed monthly cost.
+To function properly, a project needs its **"core"** to be running - which contains a logger service, a statistics service, HTTP routing with SSL certificates, IP routing with built-in firewall etc. It is now created when you create the project itself, it doesn't wait for the first service anymore.
```

Comment:
I wanted to only state that it is created right away - "It is created when you create the project itself" is fine. Since most people don't know how it worked before, it is not necessary to explain.